app/code/community/Inviqa/SymfonyContainer/Model/Observer.php

```diff
@@ -12,9 +12,15 @@ class Inviqa_SymfonyContainer_Model_Observer
     public function onCacheRefresh(Varien_Event_Observer $event)
     {
         if (ConfigurationBuilder::MODEL_ALIAS === $event->getType()) {
-            $filePath = Mage::getBaseDir('cache') . '/' . ConfigurationBuilder::CACHED_CONTAINER;
-            if (file_exists($filePath)) {
-                unlink($filePath);
+            $containerFilePath = Mage::getBaseDir('cache') . '/' . ConfigurationBuilder::CACHED_CONTAINER;
```

Comment:
It does, @jon-acker, but you read _a file_ (or its properties), whereas I said when you read a _path_ that the system gives _you_.